app/assets/javascripts/admin_layout.js

```diff
@@ -1,5 +1,7 @@
+//= require jquery/dist/jquery
 //= require govuk_publishing_components/dependencies
 //= require govuk_publishing_components/all_components
+//= require components/autocomplete
 
 //= require jquery-ui.sortable.min
 //= require jquery_ujs
```

Comment:
```suggestion

```

Reply:
Hmm just tried that and I'm still getting errors from missing jQuery.